Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Production.xml

```diff
@@ -0,0 +1,151 @@
+		<interactionCellOffset>(0,0,-1)</interactionCellOffset>
+		<surfaceType>Item</surfaceType>
+		<tickerType>Normal</tickerType>
+		<comps>
+			<li Class="FluffyManager.CompProperties_ManagerStation">
+				<compClass>FluffyManager.Comp_ManagerStation</compClass>
+				<speed>1000</speed>
+			</li>
+		</comps>
+	</ThingDef>
+
+	<ThingDef ParentName="BenchBase">
+		<DefName>FM_ManagerStation</DefName>
+		<label>manager desk</label>
+		<ThingClass>FluffyManager.Building_ManagerStation</ThingClass>
+		<Description>A desk with a computer and all the stationary required to allow a manager to do his work efficiently.</Description>
+		<graphicData>
+			<texPath>Buildings/ManagerStation</texPath>
+			<graphicClass>Graphic_Multi</graphicClass>
+			<shaderType>CutoutComplex</shaderType>
+			<drawSize>(3.5,1.5)</drawSize>
+		</graphicData>
+		<costStuffCount>100</costStuffCount>
+		<costList>
+			<SteelBar>50</SteelBar>
+			<Electronics>25</Electronics>
+		</costList>
+		<stuffCategories>
+			<li>Metallic</li>
+			<li>Woody</li>
+			<li>Stony</li>
+		</stuffCategories>
+		<AltitudeLayer>Waist</AltitudeLayer>
+		<fillPercent>0.5</fillPercent>
+		<useHitPoints>True</useHitPoints>
+		<statBases>
+			<WorkToMake>3000</WorkToMake>
+			<MaxHitPoints>180</MaxHitPoints>
+			<Flammability>1.0</Flammability>
+		</statBases>
+		<Size>(3,1)</Size>
+		<DesignationCategory>Misc</DesignationCategory>
+		<Passability>PassThroughOnly</Passability>
+		<pathCost>60</pathCost>
+		<hasInteractionCell>True</hasInteractionCell>
+		<interactionCellOffset>(0,0,-1)</interactionCellOffset>
+		<surfaceType>Item</surfaceType>
+		<tickerType>Normal</tickerType>
+		<comps>
+			<li Class="CompProperties_Power">
+				<compClass>CompPowerTrader</compClass>
+				<basePowerConsumption>100</basePowerConsumption>
+				<startElectricalFires>true</startElectricalFires>
+				<soundPowerOn>PowerOnSmall</soundPowerOn>
+				<soundPowerOff>PowerOffSmall</soundPowerOff>
+			</li>
+			<li Class="FluffyManager.CompProperties_ManagerStation">
+				<compClass>FluffyManager.Comp_ManagerStation</compClass>
+				<speed>500</speed>
+			</li>
+		</comps>
+		<researchPrerequisites>
+			<li>ManagingSoftware</li>
+		</researchPrerequisites>
+	</ThingDef>
+
+	<ThingDef ParentName="BenchBase">
+		<DefName>FM_AIManager</DefName>
+		<label>AI manager</label>
+		<ThingClass>FluffyManager.Building_AIManager</ThingClass>
+		<Description>An enormous supercomputer capable of autonously managing your colony. Claims that the AI has a malevolent psychopath subroutine are entirely false. We promise.</Description>
+		<graphicData>
+			<texPath>Buildings/AI_ManagerStation</texPath>
+			<graphicClass>Graphic_Multi</graphicClass>
+			<shaderType>CutoutComplex</shaderType>
+			<drawSize>(2.5,2.5)</drawSize>
+		</graphicData>
+		<costList>
+			<PlasteelBar>75</PlasteelBar>
+			<AIPersonaCore>1</AIPersonaCore>
+			<IntegratedCircuitsBoard>25</IntegratedCircuitsBoard>
+		</costList>
+		<AltitudeLayer>Waist</AltitudeLayer>
```

Comment:
<AltitudeLayer>Waist</AltitudeLayer> is not there anymore in a14 and gives an error
Did you get the def from the modpack a13 then? Cause there is an a14 release of the mod where fluffy would have changed this I believe